src/js/dash/segments/getSegmentListForRepresentation.js

```diff
@@ -131,7 +144,7 @@ createSegmentListFromTemplate = function(representationXml) {
                     initializationRelativeUrl = segmentTemplate.replaceIDForTemplate(initializationRelativeUrlTemplate, representationId);
 
                 initializationRelativeUrl = segmentTemplate.replaceTokenForTemplate(initializationRelativeUrl, 'Bandwidth', representationXml.getBandwidth());
-                return baseUrl + initializationRelativeUrl;
+                return [baseUrl, initializationRelativeUrl].join('/');
```

Comment:
I'm not sure :) I did a couple quick fixes to get the demo working but a more comprehensive review of a lot of the MPD spec and the handling in this project should be on our TODOs.